docs/meshcloud.project.md

```diff
@@ -76,4 +76,26 @@ In the **Project Users** section you can click the `-` button in the row of a us
 
 ## Delete a meshProject
 
-If you would like to delete a project which is no longer used, go to your **Account** settings. Navigate to **Projects** and click the button with the trash icon. The system now checks whether any resources still exist for your project. If there are existing resources, you will be informed about which resources still exist. As deletion of a project is an irrevocable action, you cannot delete projects with resources. You have to delete those resources manually before to be sure that you really want to delete these resources that may contain production relevant workloads. When no resources exist anymore you have to confirm the deletion by entering the identifier of your project again. The actual project deletion in the platforms will be done in the background and may take a while.
+If you would like to delete a project which is no longer used, go to your **Account** settings. Navigate to **Projects** and click the button with the trash icon.
+
+When you want to delete a meshProject it is not completely removed in the cloud platforms and in the database, only flagged for deletion.
+
+The deletion procedure depends on the variaty of meshTenants under the project:
+1. a project contains exclusively public cloud meshTenats (AWS, Azure, GCP)
```

Comment:
It's not only about public cloud tenants though. It's about the platforms for which we don't have resource existence check. Right now Kubernetes and Openshit also fall under this category. I think the title 1. should also be changed to capture those platforms.

Reply:
You are right! I'll change this